diff --git a/node_modules/@types/file-saver/index.d.ts b/node_modules/@types/file-saver/index.d.ts
index fb291a3..d122286 100644
--- a/node_modules/@types/file-saver/index.d.ts
+++ b/node_modules/@types/file-saver/index.d.ts
@@ -34,12 +34,4 @@ declare namespace FileSaver {
     function saveAs(data: Blob | string, filename?: string, disableAutoBOM?: boolean): void;
 }
 
-declare global {
-    const saveAs: typeof FileSaver.saveAs;
-
-    interface Window {
-        saveAs: typeof FileSaver.saveAs;
-    }
-}
-
 export = FileSaver;
