diff --git a/node_modules/@webtoon/psd/dist/classes/Psd.d.ts b/node_modules/@webtoon/psd/dist/classes/Psd.d.ts
index 3f7c9a0..d0cb67d 100644
--- a/node_modules/@webtoon/psd/dist/classes/Psd.d.ts
+++ b/node_modules/@webtoon/psd/dist/classes/Psd.d.ts
@@ -9,7 +9,7 @@ import { Synthesizable } from "./Synthesizable";
  * A parsed PSD file.
  * @alpha
  */
-export declare class Psd extends Synthesizable implements NodeBase<never, NodeChild> {
+export declare class Psd extends Synthesizable implements NodeBase<undefined, NodeChild> {
     private parsingResult;
     readonly name = "ROOT";
     readonly type = "Psd";
